lib/custom_html_renderer.rb

```diff
@@ -0,0 +1,7 @@
+require 'middleman-core/renderers/redcarpet'
+
+class CustomHTMLRenderer < Middleman::Renderers::MiddlemanRedcarpetHTML
+  def image(*args)
+    %(<a href="#{args.first}" target="_blank">#{super}</a>)
```

Comment:
I'm not sure, that's a valid point. I copied the behaviour from GitHub, which does open in it a new tab, but I suppose that's probably worth putting up for discussion.